### PR TITLE
Add server quick info to help debug container

### PR DESCRIPTION
**Pull request by chez14**
In this PR:
 - Add new Discord commang `!stats` to show current bound twitter user and package version info.

This hopefully will help us to show current version of installed package on the Docker realm, #3.

Relates with #2

### CODE REVIEW

**Comment by chez14** [comment]
@Namchee, please help me review this one ya. Do this meets your coding standard? Thanks!

File: src/discord/commands/stats.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  description: 'Show current fleet info such as version.',
+  params: [],
+  example: `\`${prefix}stats\``,
+  execute: async (
+    message: Message,
+    _: string[],
+    { twitterRepository }: Context,
+  ): Promise<Message> => {
+    const { channel } = message;
+    let userInfo = await twitterRepository.getSelfInfo();
```
Comment (Namchee) [inline src/discord/commands/stats.ts:17]: Should be `const`

File: src/discord/commands/stats.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    let reply = [
```
Comment (Namchee) [inline src/discord/commands/stats.ts:19]: Also `const`

File: src/discord/commands/stats.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      `App run on ${name}@v${version}.`,
+      `Twitter Account bound on \`${userInfo?.name} @${userInfo?.screen_name}\``,
+      `https://twitter.com/${userInfo?.screen_name}`
+    ];
+
+    return channel.send(reply.join("\n"));
```
Comment (Namchee) [inline src/discord/commands/stats.ts:25]: I think it's better with `MessageEmbed`

[Reference](https://discordjs.guide/popular-topics/embeds.html)

File: src/repository/twitter.ts [hunk 4]
```diff
@@ -107,4 +107,22 @@ export class TwitterRepository {
...
   }
+
+  /**
+   * Get information regarding current authenticated tweet account.
+   *
+   * @return {Promise<any>} user profile, and null when it fails.
+   */
+  public getSelfInfo = async (): Promise<any> => {
+    try {
+      let response = await this.twitterClient.get(
```
Comment (Namchee) [inline src/repository/twitter.ts:118]: `const`

File: src/repository/twitter.ts [hunk 5]
```diff
@@ -107,4 +107,22 @@ export class TwitterRepository {
...
+        'statuses/user_timeline',
+        { count: 1, include_rts: true, exclude_replies: false }
+      );
+
+      return response?.[0]?.user;
```
Comment (Namchee) [inline src/repository/twitter.ts:123]: Probably use destructuring? `?` somehow made me anxious

**Review by Namchee — CHANGES_REQUESTED** [review]
I'll leave some comments here. Mostly about `let` and `const`. Follow the linter rules.

**Comment by chez14** [comment]
Hey @Namchee,

I've updated accordingly, just quick request, can you please add the es-lint runner to `package.json`. I intended to run the eslint **check** upon PR creation, so things can get attention accordingly.

Thanks!

**Comment by Namchee** [comment]
@chez14,

I've added some linter commands in the `package.json`. Please merge it first and push it back so I can merge this one :smiley:

**Comment by chez14** [comment]
Done. I'm rebasing the branch just to make sure.

**Review by Namchee — APPROVED** [review]

**Comment by repurika** [comment]
:rocket: PR was released in [`v0.1.0`](https://github.com/Namchee/daily-watori/releases/tag/v0.1.0) :rocket:

**Pull request merged**